samples/tfm_hello_world/README.rst

```diff
@@ -0,0 +1,26 @@
+.. _tfm_hello_world:
+
+TF-M Hello World
+################
+
+Overview
+********
+
+A simple sample based on Hello World that demonstrates adding TF-M to an
+application.
+
+Building and Running
+********************
+
+This application works on the nRF9160 DK and the nRF5340 DK/PDK. Use the ns
+variants of the boards.
+
+Sample Output
+=============
+
+.. code-block:: console
+
+    Hello World! nrf5340pdk_nrf5340_cpuapp
```

Comment:
```suggestion
    Hello World! nrf5340pdk_nrf5340_cpuappns
```

Reply:
The output actually shows as specified. Not sure why.